content/docs/04-clusters/06-cluster-management/03-compliance-scan.md

```diff
@@ -37,41 +39,93 @@ The scan summary of total passed and failed tests are displayed while the test i
+* Pod
+  * Scan all images used by a single **Pod**
+
+## Viewing SBOM scan results
+You can click into a completed scan to view a scan report containing additional detail for every image that was scanned. The context column indicates every unique usage of each image, broken out by container name, namespace, and pod name, as each image may be in use by various containers within a given scope. The vulnerability summary column provides a condensed view of the vulnerability report, which can be viewed in greater detail by clicking on any row in the scan report.
 
-Cluster scan of any type can be started by navigating to the scans tab of a cluster on the UI. The scan shows up as 'initiated' and transitions to 'Completed' when the scan is complete.
+![sbom_results.png](/sbom_results.png)
+
+Each image details page within the scan report provides a list of dependencies and vulnerabilities. These tables are condensed highlights of the metadata contained in the SBOM that was generated for a particular image. Each dependency’s version and type is displayed, but additional metadata will be included in the SBOM. Exactly what additional metadata is included will depend on the selected SBOM format.
+
+![sbom_dependencies.png](/sbom_dependencies.png)
+
+For each vulnerability, you can view the name, severity level, vulnerability code, installed or impacted version, and the fix version (if a fix is available). Any CVEs documented in the [NIST National Vulnerability Database](https://nvd.nist.gov/vuln) (NVD) will render as a hyperlink to the NVD detail page for that particular vulnerability.
+
+![sbom_vulnerabilities.png](/sbom_vulnerabilities.png)
+
+# Scan Options
+
+The following options are available for running cluster scans:
+   
+## On Demand
+A cluster scan of any type can be started by navigating to the scans tab of a cluster on the UI. The scan shows up as 'initiated' and transitions to 'Completed' when the scan is complete.
 
 |__On Demand Scan__|
 |------------------|
 |Select the cluster to scan -> Scan(top panel) -> Run Scan|
 
-
-
 ## Scheduled
-
 A schedule can be set for each type of scan at the time of deploying the cluster initially. The schedule can also be (re)set at a later point.
 
 |__During Cluster Deployment__|
 |-----------------------------|
 |Add New Cluster -> Cluster Policies -> Scan Policies -> Enable and schedule desired scans|
 
-
 |__Running Cluster__|
 |----------------------|
 |Select the cluster to scan -> Settings -> Cluster Settings -> Scan Policies -> Enable and schedule scans of your choice|
 
-
 ### Schedule Options Available
 
-* Custom your compliance scan for an exact month, day, hour, and minute of user choice
-* Every week on Sunday at midnight
-* Every two weeks at midnight
-* Every month on the 1st at midnight
-* Every two months on the 1st at midnight
+* Schedule your compliance scan for an exact month, day, hour, and minute
```

Comment:
```suggestion
Schedule your compliance scan for month, day, hour, or minute. For example: 
* Every week on Sunday at midnight.
* Every two weeks at midnight.
* Every month on the first day of the month at midnight.
* Every two months on the first day of the month at midnight
```

Reply:
Made the examples a bullet list and removed the top-level bullet.

It would be useful to include an example of hour and minute to replace one of the week and one of the month examples.